src/app/catalog/models/user-embeddable.ts

```diff
@@ -0,0 +1,4 @@
+export type UserEmbeddable = {
```

Comment:
if there is no wery impoertant reasons to keep this name I would  use UserShortInfo
and I would use this
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
done